README.md

```diff
@@ -50,8 +50,9 @@ Creates a new instance of the Zenfi SDK.
 |`partnerName`|String|The name of the platform which implements this code. This parameter is required to send events to the Zenfi API.|
 |`targets`|[Object]|The list of html elements and their configuration to be filled.|
 |`targets.dataKey`|String|The name of the key in the `leadInfo` object which contains the information to be filled.|
-|`targets.selector`|String|The css selector to find the element in the DOM.|
-|`targets.strategy`|String|Indicates the type of strategy to fill the DOM element. Possible values are: `value` (sets the value property of an input), `html` (Puts the content inside the HTML tag, used for `<div />`s). If no `strategy` is specified, it will try the `value` strategy by default.|
+|`targets.selector`|String \| Function|The css selector to find the element in the DOM. If a function is passed, it will receive an object with `{ dataKey, strategy, value }` properties and must return a string with the css selector.|
```

Comment:
Fíjate que me tardé un rato en parsear mentalmente que recibía los datos por parámetro hasta que vi el ejemplo. 

Estaba pensando en estas tres alternativas por si crees que vale la pena:

La primera es que el tipo especifique los parámetros:

```suggestion
|`targets.selector`|String \| Function({ dataKey, strategy, value })|The css selector to find the element in the DOM. If a function is passed, it will receive an object with `{ dataKey, strategy, value }` properties and must return a string with the css selector.|
```

La segunda es sólo especificar que es por parámetro:

```suggestion
|`targets.selector`|String \| Function|The css selector to find the element in the DOM. If a function is passed, it will receive an object with `{ dataKey, strategy, value }` as param properties and must return a string with the css selector.|
```

La última es proveer un valor de ejemplo, un poco inspirado de docs que he visto como [esta](https://github.com/wojtekmaj/react-calendar#calendar), aunque reconozco que el ejemplo está abajo

```suggestion
|`targets.selector`|String \| Function|The css selector to find the element in the DOM. If a function is passed, it will receive an object with `{ dataKey, strategy, value }` properties and must return a string with the css selector.| ({ dataKey, strategy, value }) => { ... }
```

Reply:
+1
Tomé la segunda opción, porque las otras movía mucho el layout de la tabla.